assets/components/src/preview-box/index.js

```diff
@@ -10,8 +10,8 @@ import './style.scss';
 
 const PreviewBox = ( { children } ) => (
 	<div className="newspack-preview-box">
-		<span className="newspack-preview-box__label">{ __( 'Preview', 'newspack' ) }</span>
-		<span className="newspack-preview-box__content">{ children }</span>
+		<div className="newspack-preview-box__label">{ __( 'Preview', 'newspack' ) }</div>
```

Comment:
Don't see this component in the demo page—should we add it?

Reply:
It's only used in 1 place, the SEO wizard. I don't think it's worth adding to the demo page.